applications/serial_lte_modem/prj.conf

```diff
@@ -101,6 +101,7 @@ CONFIG_SLM_START_SLEEP=n
 CONFIG_SLM_DATAMODE_URC=n
 
 # Debug configurations
+#CONFIG_SLM_LOG_LEVEL_DBG=y
```

Comment:
This is debug code

Reply:
That is debug code, and there are also other usually used debug configurations below that. Note that it is commented out. I would like to keep those in there as they speed debugging. Presumably for customers as well.

Reply:
CONFIG_SLM_LOG_LEVEL_INF=y already exists, can change that to _DBG for debug level
Otherwise there'd be two log levels for SLM in one prj.conf

Reply:
Given that log level is INF by default, the CONFIG_SLM_LOG_LEVEL_INF=y actually does nothing. Unless you are very much against removing it, I would suggest that I remove that one and leave the debug one in comments.

Reply:
As you said Zephyr logging default level 3 is level INF.
Well I'm OK with either way.